public/static/docs/command-reference/import.md

```diff
@@ -129,8 +128,8 @@ outs:
 
 Several of the values above are pulled from the original stage file
 `model.pkl.dvc` in the external DVC repository. The `url` and `rev_lock`
-subfields under `repo` are used to save the origin and version of the
```

Comment:
not a big deal, but I think it's fine I think to use version term here - that's the essence of it from a user perspective

Reply:
Except that the field is called `rev_lock` so using "revision" helps understand what "rev" stands for, I think. But in general I've reviewed all my previous changes to minimize "revision" in favor of "version" or other general wording now.

Reply:
since it's `url + rev` it's fine to to use a higher level term - version

Reply:
ah, I see you changed that ... I still prefer the human friendly version to be honest. Feels like we complicate this w/o a good reason.

Reply:
Yes, this was some time ago but in hindsight I also like "version" better. Changed back.